orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/client/SaltConnector.java

```diff
@@ -133,6 +133,7 @@ public GenericResponse health() {
     @Measure(SaltConnector.class)
     @Retryable(value = ClusterProxyWebApplicationException.class, backoff = @Backoff(delay = 1000))
     public GenericResponses pillar(Iterable<String> targets, Pillar pillar) {
+        LOGGER.debug("Executing salt pillar targets: {}, pillar: {}", targets, pillar);
```

Comment:
Do you have an example log line? Pillars could have sensitive data in them like DB passwords.

Reply:
Unfortunately I don't have example log line but I agree they can contain sensitive data in this case do you suggest to keep just `targets` in log and eliminating `pillar`?

Reply:
@keyki  I have eliminated the pillar.

Reply:
Thanks.